template.yaml

```diff
@@ -433,6 +438,7 @@ Resources:
           BY_PUBLISHED_PUBLICATIONS_INDEX_NAME: !Ref PublishedPublicationsIndexName
           API_SCHEME: https
           API_HOST: !Ref CustomDomain
+          AWS_REGION: ${AWS::REGION}
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Found an embedded parameter outside of an &#34;Fn::Sub&#34; at Resources/NvaListPublicationsFunction/Properties/Environment/Variables/AWS_REGION